frontend/docs/user-guide/configuration.md

```diff
@@ -0,0 +1,63 @@
+---
+outline: deep
+---
+
+# Configuration
+
+Users with Administrator permission can access {{ $params.APPLICATION_NAME }}’s Configuration page. This page is divided into three sections:
+
+1. Dynamic Configuration
```

Comment:
I think we could add [MD heading links here like a table of contents](https://www.markdownguide.org/extended-syntax/#linking-to-heading-ids)